src/components/Grid/Grid.tsx

```diff
@@ -13,6 +13,16 @@ export interface GridProps extends React.HTMLAttributes<HTMLElement> {
   element?: React.ElementType<GridProps>;
 }
 
+/**
+ * ## Overview
+ *
+ * A wrapper component for adding a 12-column grid layout.
+ *
+ * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=6226-1330&m=dev)
```

Comment:
I think this is a more direct link and should be used on GridItem as well.

```suggestion
 * [Figma Link](https://www.figma.com/design/xMuOXOAKVt5HC7hgYjF3ot/Components-v2.0?node-id=7274-9908&m=dev)
```